src/Native/System.Native/pal_io.cpp

```diff
@@ -359,6 +359,39 @@ int32_t FcntlSetPipeSz(int32_t fd, int32_t size)
 }
```

Comment:
Sorry I wasn't very clear... What I meant was maybe this could be placed in a folder like System.Native.Unix instead of System.Native. This way the intent is very clear.